tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -137,9 +137,9 @@ def stop_node_cb():
         await time_out_assert(5, wallet.get_confirmed_balance, initial_funds)
         await time_out_assert(5, wallet.get_unconfirmed_balance, initial_funds)
 
-        client = await WalletRpcClient.create(self_hostname, test_rpc_port, bt.root_path, config)
-        client_2 = await WalletRpcClient.create(self_hostname, test_rpc_port_2, bt.root_path, config)
-        client_node = await FullNodeRpcClient.create(self_hostname, test_rpc_port_node, bt.root_path, config)
+        client = await WalletRpcClient.create(hostname, test_rpc_port, bt.root_path, config)
+        client_2 = await WalletRpcClient.create(hostname, test_rpc_port_2, bt.root_path, config)
+        client_node = await FullNodeRpcClient.create(hostname, test_rpc_port_node, bt.root_path, config)
```

Comment:
Do we want to be switching to pulling this out of the config?  If so, should we be using the fixture up above?

Reply:
`self_hostname` and `bt.config["hostname"]` are certain to be the same if a test imports both `bt` and `self_hostname`

Reply:
Which is the preferred method?  It would seem like the fixture would be since that would maintain consistency.  Also, it would leave these lines as they were before.  The other lines should be changed to match, but since this could all not be changed at all in this PR, I'm fine with that route.

Reply:
I think it's a mistake to make `self_hostname` a fixture. it restricts access to it to only test functions and other fixtures. Currently we use it in other places too.

It should especially not have to depend on constructing a whole `BlockTools` object.

Reply:
> Which is the preferred method? It would seem like the fixture would be since that would maintain consistency.

in `main` (that this PR is targeting), `self_hostname` is a global string literal in the tests. I don't think there's a reason for this patch to change that.